test/functionalTest/cases/1308_subscription_cache/subCacheMixedSubscriptionsWithUpdatesAndDeletes.test

```diff
@@ -32,21 +32,21 @@ accumulatorStart
 
 #
 # 01. Create subscription S1 - without pattern
-# 02. Check statistics, see NO entries in subCache
+# 02. Check statistics, see ONE entry in subCache
```

Comment:
Again, I understand this +1 is due to a isPattern=false subscription already created in SHELL. Is that correct?